src/app/toolkit/components/SortableItem.tsx

```diff
@@ -16,10 +16,20 @@ interface SortableItemProps {
   handleDelete: (id: string) => void;
 }
 
+// const isValidUrl = (url: string | undefined): boolean => {
+//   if (!url) return false;
+//   try {
+//     new URL(url);
+//     return true;
+//   } catch {
+//     return false;
+//   }
+// };
+
```

Comment:
```suggestion
// const isValidUrl = (url: string | undefined): boolean => {
//   if (!url) return false;
//   try {
//     new URL(url);
//     return true;
//   } catch {
//     return false;
//   }
// };
```

Do we need to add this commented out code? [will not block merge]